.github/workflows/tests.yml

```diff
@@ -494,6 +494,240 @@ jobs:
           name: lxd-clients-${{ runner.os }}
           path: bin/
 
+  ui:
+    name: UI e2e tests
+    runs-on: ubuntu-latest
+    needs: [code-tests, documentation]
+    if: "!startsWith(github.ref, 'refs/heads/stable-')"
+    env:
+      LXD_DIR: "/var/lib/lxd"
+      LXD_OIDC_CLIENT_ID: "gxj297yfAjmklILK5WqPWDSbtVBAeSQm"
```

Comment:
Is this a secret key?

Reply:
Same as the [password over there](https://github.com/canonical/lxd/pull/14035#discussion_r1841906916)

Reply:
As per the [okta docs](https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/), the client id is public.